### ISSUE TITLE
Update help for the new SP

### ISSUE THREAD

**Issue by adam-collins** [AtlasOfLivingAustralia/spatial-hub#128]
Update the help information on the Spatial Portal to reflect the changes in the new Spatial Portal.

Details on [Confluence page](https://confluence.csiro.au/display/ALASD/Spatial+portal%2C+version+2.0+-+Release+notes+and+documentation+changes)

**Sprint 1: Oct 2-12**

Goals for this sprint:

1. Understand the extent of the changes from when the existing help documentation was written (it would be good to know what version of the spatial portal that was) to the new version of the spatial portal that will soon be ready for release (version x.x).
2. Create a set of notes outlining the changes (these should be able to take the form of release notes which will help both our external customers and internal staff needing to work on the spatial portal)
3. Determine whether the existing help documentation will continue to be relevant or will need to be rewritten as it is no longer valid & produce a list of help documentation that requires updating.

see https://github.com/AtlasOfLivingAustralia/spatial-portal/issues/153

# Tasks
- [ ] #141 - Update Help: Scatterplot redo screenshots
- [ ] #142 - Update Help: Point Comparisons help page
- [ ] #143 - Update Help: Area of Occupancy and Extent of Occurrence pages
- [ ] #144 - Update Help: Generate Points page
- [ ] #145 - Update Help: Panoramio pages
- [ ] #146 - Update Help: Facet (Add to Map)
- [ ] #147 - Update Help: Common map options
- [ ] #148 - Update Help: Scatterplot List

**Comment by adam-collins** [comment]
Include https://github.com/AtlasOfLivingAustralia/spatial-portal/issues/152

**Comment by adam-collins** [comment]
Include https://github.com/AtlasOfLivingAustralia/spatial-portal/issues/150

Note: 'Export map' uses different DPI images to the screen display so some features may differ.

**Comment by Tasilee** [comment]
From https://github.com/AtlasOfLivingAustralia/spatial-portal/issues/153

Update the help information on the Spatial Portal to reflect the changes in the new Spatial Portal.

**Sprint 1: Oct 2-12**

Goals for this sprint:
1. Understand the extent of the changes from when the existing help documentation was written (it would be good to know what version of the spatial portal that was) to the new version of the spatial portal that will soon be ready for release (version x.x).
2. Create a set of notes outlining the changes (these should be able to take the form of release notes which will help both our external customers and internal staff needing to work on the spatial portal)
3. Determine whether the existing help documentation will continue to be relevant or will need to be rewritten as it is no longer valid & produce a list of help documentation that requires updating.

**Comment by Tasilee** [comment]
From email to @RobinaSanderson 1st October 2018

There is no new functionality in the beta Spatial Portal. There are however some minor formatting differences that I am aware of
In the current version, each step of the wizards are on a separate page on a pop-up window. In the beta, all steps are on a single scroll-able page.  
The Google zoom bar has been simplified. Zoom to user location is not implemented in the beta
The rollover value (existing) versus click at location (beta) for displaying environmental/contextual layer values
Slightly different formatting of scale bar and coordinates at cursor
Header bar content and format has been changed (contact us, get involved ALA apps, save session, user settings to save session, load session, user settings, ALA home page)
Species point pop-up (beta) displays (in addition to existing version info) "List records at this point"
Display user-images on map used the now defunct Google Panoramio and now uses Flickr (but it operates exactly the same).
Adam: What have I missed?

In short, the great majority of the changes to the SP Help simply involve changing the screen captures.

**Comment by sat01a** [comment]
It appears this ticket still needs some more work. Pushing it back to TODO column.

**Comment by RobinaSanderson** [comment]
The tasks specified on this ticket above are done - however, the scope of this work is to update or rewrite the all 42 How-To articles. 23 of these are done. If there a particular ones that need earlier attention, please list them.
